### ISSUE TITLE
mermaid integration?

### ISSUE THREAD

**Issue by dharmeshkakadia** [excalidraw/excalidraw#2003]
Thanks for the excalidraw! Love it. 

I (like a lot of other people) use [mermaid](https://mermaid-js.github.io/mermaid/) for source controlling diagrams. I would love to see if there is a way to render excalidraw diagrams from mermaid definition. Does something like this exist? If not is there an API/CLI tool/workflow that can achieve this? Thanks!

**Comment by xixixao** [comment]
Sounds like what you want is https://roughjs.com/, not necessarily excalidraw. Excalidraw is a digital whiteboard, while mermaid does the drawing for you already.

**Comment by dwelle** [comment]
Yes, what you're asking for is for us to reimplement Mermaid. Much easier to make Mermaid render using roughjs instead.

So, to answer your question --- it would be impossible on our end, and an API wouldn't help you in this regard.

**Comment by dharmeshkakadia** [comment]
Thanks!
